cmd/createenvironment.go

```diff
@@ -38,7 +38,7 @@ var environmentCmd = &cobra.Command{
 	Short: "Create a new environment",
 	Long: AddAppName(`Create a new environment
     Example:
-    $AppName create environment env1 --organization org --file ./somepath.json`),
+    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
```

Comment:
Actually, this doesn't fix since it is a folder path and it expects yaml.

Reply:
You mean it should be like this?

 create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath

Reply:
yes, it may be artifactsfolder